src/_pytest/terminal.py

```diff
@@ -255,6 +251,22 @@ def _determine_show_progress_info(self):
             return False
         return self.config.getini("console_output_style") in ("progress", "count")
 
+    @property
+    def verbosity(self):
+        return self.config.option.verbose
+
+    @property
+    def showheader(self):
+        return self.verbosity >= 0
```

Comment:
This property is only used once in the codebase. Not sure if it is worth the effort to introduce this property. I would remove it and add a comment to the place where it is used. 
I doubt that we can remove it, since it is an API change, right?

Reply:
Agree, let's keep it, it is not like it is a huge burden anyway. 👍

Reply:
No, they are not really a burdern.